simplesat/sat/priority_queue.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from functools import partial
+from heapq import heappush, heappop
+from itertools import count
+
+
+class _REMOVED_TASK(object):
+    pass
+REMOVED_TASK = _REMOVED_TASK()
```

Comment:
Or maybe not. I guess it's useful to see something other than `<object>` when debugging.